src/components/NgoDetails/index.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Button } from "react-bootstrap";
+import "./index.css";
+import { useTranslation } from "react-i18next";
+
+const NgoDetails = ({ details }) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="ngoDetailsTableData">
+      <h3>{details.facilityName}</h3>
+      <p className="address">{details.address}</p>
+      <h5>{t("resources.ngoList.ngoDetails.aidDescription")}</h5>
+      <p className="ngoDetailsServiceDescription">
+        {details.serviceDescription}
+      </p>
+      <p>
+        {`${t("resources.ngoList.ngoDetails.responsiblePerson")} ${
```

Comment:
I dont think u need the backslack and dollar sign here in order to render the variables, u can jusr render it inside the curly brackets right away